tests/tests.go

```diff
@@ -209,6 +214,29 @@ func initKubemacpoolParams(rangeStart, rangeEnd string) error {
 	return nil
 }
 
+func getVmFailCleanupWaitTime() time.Duration {
+	managerDeployment := v1.Deployment{}
+	err := testClient.VirtClient.Get(context.TODO(), client.ObjectKey{Namespace: managerNamespace, Name: names.MANAGER_DEPLOYMENT}, &managerDeployment)
+	Expect(err).ToNot(HaveOccurred(), "Should successfully get manager's Deployment")
+	Expect(managerDeployment.Spec.Template.Spec.Containers).ToNot(BeEmpty(), "Manager's deployment should contain containers")
+
+	for _, container := range managerDeployment.Spec.Template.Spec.Containers {
+		if container.Name == deploymentContainerName {
```

Comment:
Let's remove a little of nesting
```golang
container.Name != deploymentContainerName {
    continue
}
```

Reply:
@qinqon @RamLavi 
 I think its not removing the nesting its just hiding it, eventually the stack depth will be the same and using if-continue in this case make the code less readable IMO

Reply:
I believe early exit (the one Quique suggested) is a good and recommended practice in go. The positive flow should be as close to the left side of the screen as possible. Ideally, you can just read the left-most block, ignoring all ifs, to get a gist on what happens in the code. That makes it more readable in my opinion.

Reply:
When looking for specific object in a collection the code in most cases, where there is no sophisticated logic, the code should be "positive" by looking for what we want `(if .. == ..`) and not what we dont (`if .. != ..`) IMO.
In this case there is no early exit when by using continue